exercises/gigasecond/src/lib.rs

```diff
@@ -1,7 +1,7 @@
-extern crate chrono;
-use chrono::*;
-
-// Returns a Utc DateTime one billion seconds after start.
-pub fn after(start: DateTime<Utc>) -> DateTime<Utc> {
-    unimplemented!()
-}
+extern crate chrono;
+use chrono::{DateTime, Duration, Utc};
```

Comment:
`Duration` is an unused import here, but when I ran `cargo check` from `exercises/gigasecond` it didn't output any warnings so I left in in since it's a useful hint. Lmk what you think of this.

Reply:
I have a mild preference to exclude `Duration` based only on an instinct that it's good for students to have to look at the docs at least a little bit. By providing the appropriate dependency crate and the stub which we have, we've already reduced the required work from the student to near nothing. 

I'm willing to be overridden on this if people feel strongly otherwise, though.

Reply:
The original angle that I wanted this exercise to take was to be some sort of introduction to crates (specifically for Rust) in addition to time/duration (applies to this exercise no matter what the language). That is what I thought in https://github.com/exercism/rust/pull/79#issuecomment-198557453. Note that at the time there was no stub at all. I may have forgotten about that when reviewing https://github.com/exercism/rust/pull/306 which added the stub. Encouraging a look at the docs seems good. As demonstrated by my having implicitly changed my mind about this a few times over the years, I don't feel strongly about it.

Reply:
I say remove it but add a note to the hints file with a link to the documentation, so students who get lost in the documentation (or don't know how to use it yet) won't get stuck.